test/integration/integration_test.go

```diff
@@ -108,6 +109,18 @@ func TestNormalOperation(t *testing.T) {
 		if errNamespace != nil {
 			t.Fatal(errNamespace)
 		}
+
+		retryErr := retry.Do(func() error {
+			_, getErr := clientset.CoreV1().Namespaces().Get(context.Background(), namespaceName, metav1.GetOptions{})
+			if !errs.IsNotFound(getErr) {
+				return errors.New("Waiting for namespace "+namespaceName+" to be deleted")
+			}
+			return nil
+		}, retry.Attempts(10), retry.MaxDelay(3*time.Second))
+
+		if retryErr != nil {
+			t.Log(retryErr)
+		}
```

Comment:
If this `cleanup` function is going to be used in most of the tests, I believe we should move it to the `main.go` file of the `integration` directory.

Reply:
I  will handle this as part of https://github.com/kubearchive/kubearchive/issues/863, which I just opened. All the tests can then use this cleanup method.